pkg/investigations/clustermonitoringerrorbudgetburn/clustermonitoringerrorbudgetburn.go

```diff
@@ -82,6 +82,13 @@ func (c *Investigation) Run(r *investigation.Resources) (result investigation.In
 		return result, r.PdClient.SilenceIncidentWithNote(notes.String())
 	}
 
+	// AI POC
```

Comment:
Thought: If we merge this and promote, we'll use this on production and bloat the notes for this alert. Maybe adding this POC to a new investigation that we toggle via the experimental flag is a better idea for now?

Reply:
Yes, let me convert this to a draft for now
I think we might want to go in a direction of running this blindly against a bunch of alerts and put in the context we have available instead of doing specific questions in investigations, something along those lines